GeneralStateTests/stExample/paygasRevert.json

```diff
@@ -54,9 +47,9 @@
             "gasLimit" : [
                 "0x061a80"
             ],
-            "gasPrice" : "0x01",
+            "gasPrice" : "0x00",
             "nonce" : "0x00",
-            "secretKey" : "0x45a915e4d060149eb4365960e6a7a45f334393093061116b197e3240065ff2d8",
+            "secretKey" : "0xaa",
```

Comment:
Would it be possible to completely omit the `secretKey` entry, or is there a lot of machinery attached to it?

Reply:
I think (not sure though, would have to look into it again) omitting the `secretKey` completely is already defined behavior and replaces the sender with a dummy sender. I am not fully happy with this `"0xaa"` workaround, long term it might make sense to have a dedicated `"isAA" : true` or something like that.

Reply:
But my preference would be to go ahead with this version for now, just to have a basic working setup.

Reply:
I like `"isAA": true` quite a bit. I'm also okay with merging as is, as long as we make a note to change it for later.

Reply:
Sounds good!